pkgs/applications/networking/cluster/linkerd/default.nix

```diff
@@ -0,0 +1,24 @@
+{ stdenv, fetchFromGitHub, buildGoModule }:
+
+buildGoModule rec {
+  pname = "linkerd";
+  version = "065dd3ec9d4b84c90383b31be16ccadd34a8ab0d";
```

Comment:
The version should be the date of the commit formatted as `yyyy-mm-dd`. Also the `pname` should be suffixed with `unstable` if you're not using a released version.

Reply:
Thanks! Fixed!